test/views/behaviours/MovementsTableViewBehaviours.scala

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.behaviours
+
+import base.FakeFrontendAppConfig
+import config.FrontendAppConfig
+import org.jsoup.nodes.Document
+import org.jsoup.select.Elements
+import play.api.libs.json._
+import viewModels.ViewMovement
+
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import scala.collection.convert.ImplicitConversions._
+
+trait MovementsTableViewBehaviours extends ViewBehaviours {
+
+  implicit val frontendAppConfig: FrontendAppConfig = FakeFrontendAppConfig()
+
+  val dateTimeFormat: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")
+
+  val day1: LocalDateTime = LocalDateTime.parse("2020-08-16 06:06:06", dateTimeFormat)
+  val day2: LocalDateTime = LocalDateTime.parse("2020-08-15 05:05:05", dateTimeFormat)
+  val day3: LocalDateTime = LocalDateTime.parse("2020-08-14 04:04:04", dateTimeFormat)
+  val day4: LocalDateTime = LocalDateTime.parse("2020-08-13 03:03:03", dateTimeFormat)
+  val day5: LocalDateTime = LocalDateTime.parse("2020-08-12 02:02:02", dateTimeFormat)
+  val day6_1: LocalDateTime
+  val day6_2: LocalDateTime
```

Comment:
Since these values are for the test setup for the test cases themselves, I think moving these here causes some confusion when looking at the test classes. I think this is called "Mystery guest" (see this [blog post](https://thoughtbot.com/blog/mystery-guest) which is geared to rail but explains the pattern a bit)